fpga_interchange/nextpnr_emit.py

```diff
@@ -39,16 +39,14 @@ def main():
     # ID = 0 is always the empty string!
     assert const_ids.get_index('') == 0
 
-    if 'global_buffers' in device_config:
-        global_buffers = device_config['global_buffers']
-    else:
-        global_buffers = []
-
+    global_buffers = device_config.get('global_buffers', [])
     disabled_routethroughs = device_config.get('disabled_routethroughs', [])
+    disabled_cell_bel_placement = device_config.get(
+        'disabled_cell_bel_placement', [])
 
-    chip_info = populate_chip_info(device, const_ids, global_buffers,
-                                   device_config['buckets'],
-                                   disabled_routethroughs)
+    chip_info = populate_chip_info(
```

Comment:
perhaps at this point it would be neater to pass `device_config` as-is to `populate_chip_info` and let it read the various fields?

Reply:
Sure, makes total sense, done